lib/krane/container_logs.rb

```diff
@@ -3,7 +3,9 @@ module Krane
   class ContainerLogs
     attr_reader :lines, :container_name
 
-    DEFAULT_LINE_LIMIT = 25
+    def self.default_line_limit
```

Comment:
I'm not sure it's worth converting to a method. It's only useful for testing, but it's not a high value test.